src/Service/Apache.php

```diff
@@ -40,21 +42,20 @@ public function getSetup()
 			],
 		];
 
-		foreach ($this->configs as $config)
-		{
-			$driver                                              = Map::getType($config->get('database.driver'));
-			$version                                             = $config->getVersion('database');
-			$this->setup[$name]['links'][]                       = "$driver-$version";
-			$this->setup[$name]['environment']['VIRTUAL_HOST'][] = $config->getDomain();
-		}
+		$driver                                              = Map::getType($config->get('database.driver'));
+		$version                                             = $config->getVersion('database');
+		$this->setup[$name]['links'][]                       = "$driver-$version";
+		$this->setup[$name]['environment']['VIRTUAL_HOST'][] = $config->getDomain();
+		$this->setup[$name]['networks'][] 				     = $this->network;
 
 		$this->setup[$name]['links']                       = array_unique($this->setup[$name]['links']);
 		$this->setup[$name]['environment']['VIRTUAL_HOST'] = implode(',', $this->setup[$name]['environment']['VIRTUAL_HOST']);
 
-		return array_merge(
-			$this->setup,
-			parent::getSetup()
-		);
+//		return array_merge(
+//			$this->setup,
+//			parent::getSetup()
+//		);
+		return $this->setup;
```

Comment:
The PhpBase abstract class had no implementation for getSetup(). There main reasons for which I have removed the parent::getSetup() is because it would throw an error if no selenium container would be attached to the apache container: https://github.com/isacandrei/virtualisation/blob/container-test/src/ServiceFactory.php#L47

I believe the virtualisation scope is to have one selenium container for each server, but in the parallel testing platform, we wish to have a configurable number of selenium containers.